src/test/cpp/gtester/AliasTest.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <vector>
+#include <gtest/gtest.h>
+#include <alias/Alias.h>
+
+using namespace stride::alias;
+
+namespace Tests {
+
+TEST(AliasDeathTest, Alias)
+{	
+	//Ruzie met death
```

Comment:
You can probably work around this by throwing an `EmptyProbabilityException` instance and then catching that with `ASSERT_THROW`. I think one of Lynn's tests do that. You might want to take a look at that.